public/stylesheets/style.css

```diff
@@ -21,3 +21,19 @@
   text-decoration: line-through;
   color: #777;
 }
+
+.individual > .title {
+  display: inline;
+}
+
+.individual {
+  display: inline;
+}
+
+.glyphicon {
+  margin-right: 5px;
+}
+
+.each-bullet {
```

Comment:
What is an `each-bullet`? :P Please consider a class name that reveals what this class is being used for.